cluster/sbin/collect-debug-artifacts.sh

```diff
@@ -436,6 +439,20 @@ function collect_on_remote {
     fi
   done
 
+  # remove all unwanted lines from logs
+  if [ ! -z "${invert_pattern}" ] ; then
+    cd "${tmp_dir}"
+    for i in `ls *.log*`
+      do
+        invert_log_file_name=$i".inv"
```

Comment:
Double quote should be around whole to be effective: "$i.inv"

Though the for loop above will mess up any spaces in file names already. You will need something like:

```
for i in `ls *.log* | sed 's/^/"/;s/$/"/'`; do
```
to surround each filename in double quotes explicitly